File: clients/Python/examples/ConsumerExample.py
from zbus import Broker, Consumer
 
broker = Broker()
broker.add_tracker('localhost:15555')


def on_message(msg, client):
    print(msg)

c = Consumer(broker, 'MyTopic')
c.on_message = on_message
c.connection_count = 2 
c.start()

File: clients/C/examples/ConsumerExample.cpp
#include "Consumer.h"  


int main_Consumer(int argc, char* argv[]) {  
	Logger::configDefaultLogger(0, LOG_INFO);  

	Broker broker("localhost:15555");

	Consumer c(&broker, "MyTopic"); 
	c.messageHander = [](Message* msg, MqClient* client, void* ctx) {
		msg->print();
		delete msg;
	};  
	c.start();  


	broker.join(); 
	return 0;
}